src/packets.c

```diff
@@ -53,7 +53,8 @@ struct rte_mbuf **pg_packets_append_blank(struct rte_mbuf **pkts,
 		if (!pkts[j])
 			continue;
 		tmp = rte_pktmbuf_append(pkts[j], len);
-		memset(tmp, 0, len);
+		if (tmp)
+			memset(tmp, 0, len);
 	}
 	return pkts;
 }
```

Comment:
`packets: check that append sucess before memset`
- typo: success
- would write: `packets: check rte_pktmbuf_append success before memset`